deploy/static/provider/aws/deploy-tls-termination.yaml

```diff
@@ -28,20 +28,20 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   labels:
-    helm.sh/chart: ingress-nginx-4.0.0-beta.2
+    helm.sh/chart: ingress-nginx-4.0.0-beta.3
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/instance: ingress-nginx
-    app.kubernetes.io/version: 1.0.0-beta.2
+    app.kubernetes.io/version: 1.0.0-beta.3
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: controller
   name: ingress-nginx-controller
   namespace: ingress-nginx
 data:
-  http-snippet: |
-    server {
-      listen 2443;
+  http-snippet:
+    server{
+      listen 2443
```

Comment:
this needs ; still I believe

Reply:
Yes. Another editing mistake from me. Will push with correction